packages/edit-site/src/components/page-patterns/rename-menu-item.js

```diff
@@ -79,7 +79,7 @@ export default function RenameMenuItem( { item, onClose } ) {
 						setIsModalOpen( false );
 						onClose();
 					} }
-					overlayClassName="edit-site-list__rename_modal"
+					overlayClassName="edit-site-list__rename-modal"
```

Comment:
Also, I wanted to say, eagle eyes here! I took me a few seconds of staring to see the typo fix 😄